docs/v4.0-draft1/resources-and-data-models/aisp/direct-debits.md

```diff
@@ -72,17 +72,16 @@ The resource requires the ReadDirectDebits permission. The resource response pay
-| CategoryPurposeCode | 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/CategoryPurposeCode ||OBCategoryPurpose1Code | | |
+| MandateIdentification | 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/MandateIdentification |Name or number assigned by an entity to enable recognition of that entity, for example, account identifier|Max35Text | | |
+| Classification | 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/Classification|Specifies the type of direct debit amount, such as fixed or variable. |OBClassification1Code |FIXE<br>USGB<br>VARI| |
+| CategoryPurposeCode | 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/CategoryPurposeCode |Specifies the category purpose. For more information see `ExternalCategoryPurpose1Code` [here](https://github.com/OpenBankingUK/External_Interal_CodeSets) |OBCategoryPurpose1Code | | |
 | FirstPaymentDate | 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/FirstPaymentDate |The date on which the first payment for a Standing Order schedule will be made. |ISODate | | |
 | FinalPaymentDate | 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/FinalPaymentDate |The date on which the final payment for a Standing Order schedule will be made. |ISODate | | |
-| Frequency | 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/Frequency |A code indicating the frequency of payment for the Standing Order. Values:<br>ADHO<br>YEAR<br>DALI<br>INDA<br>MNTH<br>QURT<br>MIAN<br>WEEK |OBFrequencyPeriodType | | |
-| PeriodType | 1..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/PeriodType |A code indicating the period type for the Standing Order. Values:<br>ADHO<br>YEAR<br>DALI<br>INDA<br>MNTH<br>QURT<br>MIAN<br>WEEK |OBFrequencyPeriodType | | |
-| CountPerPeriod | 1..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/CountPerPeriod | |int32 | |
-| PointInTimeType | 1..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/PointInTimeType |A code indicating the point in time for payment of the Standing Order. Values:<br>ADHO<br>YEAR<br>DALI<br>INDA<br>MNTH<br>QURT<br>MIAN<br>WEEK |OBFrequencyPeriodType | | |
-| PointInTime | 1..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/PointInTime |ISODateTime | | |
-| Reason| 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/Reason | |Max256Text | |
+| Frequency | 0..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/Frequency |A code indicating the frequency of payment for the Standing Order. |OBFrequency6 | | |
+| PeriodType | 1..1 |OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/Frequency/PeriodType |A code indicating the period type for the Standing Order. |OBFrequency6Code |ADHO<br>YEAR<br>DALI<br>FRTN<br>INDA<br>MNTH<br>QURT<br>MIAN<br>WEEK | |
```

Comment:
Remove ref to Standing Order from Direct Debit data table.  Also validate MandateRelatedInformation schema and Paths as we change this structure earlier today (UML and swaggers are up to date)